.github/workflows/bulk_repo_update.yml

```diff
@@ -110,28 +110,28 @@ jobs:
       - name: run script
         run: ${{ github.event.inputs.script }}
 
-      - name: setup testeng-ci
+      - name: setup pull_request_creator
         run: |
-          git clone https://github.com/edx/testeng-ci.git
-          cd $GITHUB_WORKSPACE/testeng-ci
-          pip install -r requirements/base.txt
+          git clone https://github.com/openedx/repo-tools.git
+          cd $GITHUB_WORKSPACE/repo-tools
+          pip install -e '.[pull_request_creator]'
```

Comment:
What about just installing it from PyPI?

https://pypi.org/project/edx-repo-tools/0.9.0/

Reply:
🔥

Reply:
@feanil Can you review https://github.com/openedx/repo-tools/pull/523 ? It'll just push the pull_request_creator change to pypi.